keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ApproveOldVersionTests.kt

```diff
@@ -137,6 +137,10 @@ abstract class ApproveOldVersionTests<T : KeelRepository> : JUnit5Minutests {
       )
     }
 
+    after {
+      flush()
```

Comment:
This was missing and causing the DB records to leak into tests following this one. Thanks @robfletcher for the help debugging this!